app/presentation/RoomItem/index.js

```diff
@@ -16,82 +16,78 @@ import { themes } from '../../constants/colors';
 export { ROW_HEIGHT };
 
 const attrs = [
-	'name',
-	'unread',
-	'userMentions',
-	'showLastMessage',
```

Comment:
We're removing `showLastMessage` because it'll change only when the `getSettings` run (only once when the app open), right?